packages/appstore/src/components/wallet-withdrawal/wallet-withdrawal.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
-import { useActiveWallet, useCurrencyConfig } from '@deriv/hooks';
+import { useCurrencyConfig } from '@deriv/api';
```

Comment:
replaced `useCurrencyConfig` from `@deriv/hooks` to `useCurrencyConfig` from `@deriv/api`